backend/src/main/java/com/digginroom/digginroom/service/MemberService.java

```diff
@@ -0,0 +1,31 @@
+package com.digginroom.digginroom.service;
+
+import com.digginroom.digginroom.controller.dto.MemberDuplicationResponse;
+import com.digginroom.digginroom.controller.dto.MemberSaveRequest;
+import com.digginroom.digginroom.exception.MemberException.DuplicationException;
+import com.digginroom.digginroom.repository.MemberRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class MemberService {
+
+    private final MemberRepository memberRepository;
+
+    public void save(final MemberSaveRequest request) {
+        if (isDuplicated(request.memberId())) {
+            throw new DuplicationException();
+        }
+        memberRepository.save(request.toMember());
+    }
+
+    private boolean isDuplicated(String memberId) {
+        return memberRepository.findByMemberId(memberId).isPresent();
+    }
+
+    public MemberDuplicationResponse checkDuplication(String memberId) {
+        boolean duplicated = isDuplicated(memberId);
+        return new MemberDuplicationResponse(duplicated);
+    }
```

Comment:
이 부분은 제안입니다! 반영해달라는 뜻은 아닙니다 😄

결국 `Member` 도메인 객체를 만들거라면,
처음부터 `Member` 도메인 객체를 만들어두고 사용해도 좋지 않을까요?

추가로 `isDuplicated()` 함수 인자를 `Member`로 바꾸는 건 어떨까요?
멤버 중복 검증 로직을 캡슐화 해준다는 장점이 있을 것 같아요. 
핵심 도메인인 `Member` 단위로 사용할 수 있어서 좋을 것 같습니다!

멤버 중복 검증이 필요하다 -> `MemberService.isDuplicated()` 사용
`memberId`의 중복 검증이 필요하다 -> `MemberRepository.existsByMemberId()` 사용

```suggestion
    public void save(final MemberSaveRequest request) {
        Member member = request.toMember();
        if (isDuplicated(member)) {
            throw new DuplicationException();
        }
        memberRepository.save(member);
    }

    private boolean isDuplicated(Member member) {
        return memberRepository.existsByMemberId(member.getId());
    }

    public MemberDuplicationResponse checkDuplication(String memberId) {
        boolean duplicated = memberRepository.existsByMemberId(memberId);
        return new MemberDuplicationResponse(duplicated);
    }
```